src/Confluent.Kafka/Serialization/StringSerializer.cs

```diff
@@ -26,32 +30,94 @@ public class StringSerializer : ISerializer<string>
         private Encoding encoding;
 
         /// <summary>
+        ///     Name of the encoding configuration parameter for the case of key serialization.
```

Comment:
The use of "case" reads a bit strange to a Swede like me.
"Configuration property name for key serialization encoding type" or some such is clearer to me.